### ISSUE TITLE
imagenet pretrained model in Model Zoo

### ISSUE THREAD

**Issue by Cory-M** [facebookresearch/SlowFast#283]
Hi, we're trying to load the imagenet pretrained model and inflate it to the slowonly backbone. We noticed the top1 and top5 err listed there are 23.6% and 6.8%, which doesn't look quite normal - we're wondering that, is that the error rate or the accuracy rate? and how's that calculated, did you test the average over all frames within a video, or use the inflated 3d version as a fixed backbone, and evaluate on top?
Thanks so much!

**Comment by williamberrios** [comment]
Hi @Cory-M, Were you able to load the pre-trained model from ImageNet/MVIT_B_16_CONV? If so, could you provide the code for that? I have been trying to load it but I have not found any documentation in order to load it from the .pyth file. Thanks :D
